roles/hosted_engine_setup/tasks/create_target_vm/02_engine_vm_configuration.yml

```diff
@@ -86,3 +86,6 @@
       regexp: "^\\s*PermitRootLogin"
       line: "PermitRootLogin {{ he_root_ssh_access }}"
       state: present
+  - name: Clean cloud-init configuration
+    include_tasks: ../clean_cloud_init_config.yml
+    ignore_errors: true
```

Comment:
why do we need `ignore_errors: true` here?

Reply:
We don't really need it, I'll remove that.